src/components/Table/components/SortingIndicator.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Colors } from '../../../essentials';
+import { IconProps } from '../../../icons';
+
+type Props = IconProps & {
+    direction: 'ASC' | 'DESC' | '';
+};
+
+const SortingIndicator: React.FC<Props> = ({
+    size = 20,
+    color = Colors.AUTHENTIC_BLUE_900,
+    direction,
+    ...props
+}: IconProps) => (
+    <svg color={color} width={size} height={size} viewBox="0 0 20 20" fill="none" {...props}>
+        <path
+            d="M10 15.8334L7.83494 13.3334L5.66987 10.8334L10 10.8334L14.3301 10.8334L12.1651 13.3334L10 15.8334Z"
+            fill={direction === 'DESC' ? 'currentColor' : Colors.AUTHENTIC_BLUE_200}
+        />
+        <path
+            d="M10 4.16663L7.83494 6.66663L5.66987 9.16663L10 9.16663L14.3301 9.16663L12.1651 6.66663L10 4.16663Z"
+            fill={direction === 'ASC' ? 'currentColor' : Colors.AUTHENTIC_BLUE_200}
+        />
```

Comment:
Don't we have icons for this already?
I'm asking because if in the future we change arrow icons and we want to see it reflected here too we'd need to change it twice.

Reply:
While we maybe could build this with existing icons, it would be very hacky (cause we would need to overlap the bounding box to get this exact visual).

Also, this is a special icon, and shouldn't follow the other arrows necessarily. It was designed specifically for this component/use-case.

<img width="1231" alt="Screenshot 2023-04-04 at 09 52 32" src="https://user-images.githubusercontent.com/539801/229725357-be80d7cb-3706-4e16-a3b8-427cb63e9e79.png">